client/app/components/faq/faqlist.js

```diff
@@ -35,94 +30,97 @@ class FaqList extends Component {
 
 	componentDidMount() {
 		this.fetchData()
+			// eslint-disable-next-line no-console
 			.then(() => console.log("Finished mounting!"))
+			// eslint-disable-next-line no-console
```

Comment:
It's fine to leave these here, consoling is generally bad and should be explicitly stated.